modules/nodes-nodes-grace-periods-graceful-shutdown.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-grace-periods-graceful-shutdown_{context}"]
```

Comment:
I think this section has duplicate content with previous section (id="nodes-nodes-activating-graceful-shutdown_{context}") . Both said creating KubeletConfig to configure shutdownGracePeriod.

Can we remove this section or merge it's content to previous section?

Reply:
Removed.

Reply:
Migrated some content to previous topic.